bindings/nodejs/modules/open-intent/lib/chatbot-api/user-defined-actions/vm-driver.js

```diff
@@ -42,16 +42,16 @@ var Q = require('q');
 var EventEmitter = require('events');
 
 var script = "\
-commandEmitter.on('command', function(actionId, intentVariables, sessionId, next) { \
+commandEmitter.on('command', function(actionId, intentVariables, sessionId, next, storage) { \
```

Comment:
I think the storage module is not under the responsibility of the chatbot. Indeed, with the refactoring of the open-intent npm module the VM now keeps a state so that the user can create and keep a connection to redis (or any storage system) directly in the user command script. This is the responsiblity of the user to choose whether he needs a storage and the system he wants to use (redis or other).

What do you think about providing the RedisStorage as a helper of the open-intent library with an example in the documentation instead of being published in the public API (as a parameter of the user commands)?